container/openejb-loader/src/main/java/org/apache/openejb/loader/SystemInstance.java

```diff
@@ -263,7 +318,7 @@ public ClassLoader getClassLoader() {
         return removed;
     }
 
-    private static final AtomicReference<SystemInstance> system = new AtomicReference<SystemInstance>();
+    private static final AtomicReference<SystemInstance> SYSTEM = new AtomicReference<SystemInstance>();
```

Comment:
Could you use a diamond operator?
`private static final AtomicReference<SystemInstance> SYSTEM = new AtomicReference<>();`

Reply:
:) missed that changing it now